src/main/java/baritone/process/CustomGoalProcess.java

```diff
@@ -89,7 +89,8 @@ public PathingCommand onTick(boolean calcFailed, boolean isSafeToCancel) {
                     return new PathingCommand(this.goal, PathingCommandType.CANCEL_AND_SET_GOAL);
                 }
                 if (this.goal == null || (this.goal.isInGoal(ctx.playerFeet()) && this.goal.isInGoal(baritone.getPathingBehavior().pathStart()))) {
-                    onLostControl(); // we're there xd
+                    onLostControl();// we're there xd
```

Comment:
why?

Reply:
I tried something and was too stupid to enter the space there 🤦‍♂️